radio/src/targets/nv14/board.cpp

```diff
@@ -191,6 +198,14 @@ void boardOff()
     WDG_RESET();
   }
  
+ 
+#if defined(RTC_BACKUP_RAM)
+  RCC_AHB1PeriphClockCmd(RCC_AHB1Periph_BKPSRAM, DISABLE);
+  PWR_BackupRegulatorCmd(DISABLE);
+#endif
+
+  RTC->BKP0R = SHUTDOWN_REQUEST;
```

Comment:
I think this one has no effect if the peripheral clock is already disabled. This should inside the RTC_BACKUP_RAM, before the clock is disabled.